solutions/fully-configurable/catalogValidationValues.json.template

```diff
@@ -3,5 +3,5 @@
   "region": "us-south",
   "resource_tags": $TAGS,
   "name": $PREFIX,
-  "resource_group_name": $PREFIX
+  "existing_resource_group_name": $PREFIX
```

Comment:
This is going to fail because `$PREFIX` won't be an existing resource group. Maybe update to `geretain-test-resources` which is a permanent group in our account.